src/middlewares/isProfileUpdated.js

```diff
@@ -0,0 +1,12 @@
+
+const isProfileUpdated = async (req, res, next) => {
+  const { isUpdated } = req.user;
+  if (!isUpdated) {
+    return res.status(403).json({
+      status: 403,
+      error: 'Please update your profile information to continue'
+    });
+  }
+  next();
+};
```

Comment:
@hakoemmy @blaise82 Whos is supposed to internationalize this?

Reply:
@hakoemmy just created that function I think he is supposed to do it
probably he just forgot

Reply:
@RusimbiP  thank you and allow me to do it

Reply:
@RusimbiP I've implemented this, huge thanks again.